changelog/pending/20241026--engine--make-plugin-downloads-installation-cancellable-fixing-17594.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: fix
+  scope: engine
+  description: "Make plugin downloads/installation cancellable, fixing #17594."
```

I think our release process automatically adds the PR to the changelog entries

```suggestion
  description: "Make plugin downloads/installation cancellable"
```